engine/src/conversion/analysis/fun/mod.rs

```diff
@@ -952,9 +952,19 @@ impl<'a> FnAnalyzer<'a> {
                         Virtualness::PureVirtual => MethodKind::PureVirtual(receiver_mutability),
                     }
                 };
-                // Disambiguate overloads.
-                let rust_name = predetermined_rust_name
-                    .unwrap_or_else(|| self.get_overload_name(ns, type_ident, rust_name));
+
+                let rust_name = if matches!(
+                    fun.special_member,
+                    Some(SpecialMemberKind::GeneratedAccessor)
+                ) {
+                    // TODO: We can do better than this. We should be able to determine the struct name and strip that prefix
+                    (&rust_name[rust_name.find("get_").unwrap()..]).to_string()
+                } else {
+                    // Disambiguate overloads.
+                    predetermined_rust_name
```

Comment:
I wonder if you should do this irrespective, in case `get_foo` to get a field `foo` conflicts with an existing method called `get_foo`

Reply:
(though I can see you're trying to avoid such conflicts elsewhere. Hmm. That's probably fine, then)